### ISSUE TITLE
Quest book query failure 

### ISSUE THREAD

**Issue by kristofbolyai** [Wynntils/Wynntils#358]
```[19:56:10] [Render thread/INFO] (Minecraft) [CHAT] Your class has automatically been selected. Use /class to change your class, or /toggle autojoin to turn this feature off.
[19:56:10] [Render thread/INFO] (Minecraft) [CHAT] 
[19:56:10] [Render thread/INFO] (Minecraft) [CHAT]                  §bSee yourself live on the web map: 
[19:56:10] [Render thread/INFO] (Minecraft) [CHAT]                            §d§nmap.wynncraft.com
[19:56:10] [Render thread/INFO] (wynntils) Scheduling character info query
[19:56:10] [Render thread/INFO] (wynntils) Entering world WC28
[19:56:10] [Render thread/INFO] (wynntils) Scheduling quest book query
[19:56:10] [Render thread/INFO] (wynntils) Requesting rescan of Quest Book
[19:56:11] [Render thread/INFO] (wynntils) Deducing character CharacterInfo[classType=Warrior/Knight, reskinned=false, level=105, id=-1]
[19:56:11] [Render thread/WARN] (wynntils) Error querying Quest Book:Container contents without associated container open
```

This does not seem to happen every time.

**Comment by kristofbolyai** [comment]
@magicus

**Comment by magicus** [comment]
This is not as much an "error" as a report that the fail-safe mechanism of the container query is working.
I should maybe rephrase it so it does not say "error".

**Comment by kristofbolyai** [comment]
The error is that the container stays open and visible.

**Comment by magicus** [comment]
That is bad, though. :(

**Comment by magicus** [comment]
I wonder what I can do about that. The error is raised if we get a ContainerSetContentEvent before we got the MenuOpenedEvent. Maybe they can come out of order.

**Comment by magicus** [comment]
Hm.
